src/__tests__/components/Accordion.spec.js

```diff
@@ -0,0 +1,46 @@
+import '@testing-library/jest-dom'
+import { fireEvent, render, screen } from '@testing-library/react'
+import { Accordion } from '../../components'
+import { MuiThemeProvider } from '../../test-helper'
+
+describe('Accordion', () => {
+  const getRequiredProps = (override) => ({
+    content: 'contents',
+    header: 'header',
+    ...override,
+  })
+
+  describe('Given the Accordion has rendered', () => {
+    test('it should show contents', () => {
+      const props = getRequiredProps({
+        isOpenByDefault: false,
+      })
+      render(
+        <MuiThemeProvider>
+          <Accordion {...props} />
+        </MuiThemeProvider>
+      )
+
+      expect(screen.queryByText(props.header)).toBeVisible()
+      expect(screen.queryByText(props.content)).not.toBeVisible()
+
+      fireEvent.click(screen.queryByText(props.header))
+
+      expect(screen.queryByText(props.content)).toBeVisible()
+    })
+
+    test('it should show accordion content', () => {
```

Comment:
similar comment here, wondering if we should add to the test case name for clarity

```suggestion

    test('when the accordion is open by default, should show content', () => {
```